lib/schemapack.js

```diff
@@ -251,7 +252,17 @@ class SchemaPack {
       }
     });
 
-
+    if (openapi.security) {
+      authHelper = schemaUtils.getAuthHelper(openapi, openapi.security);
+      if (authHelper) {
+        if (authHelper.type === 'apiKey') {
+          if (authHelper.properties.in === 'header' || authHelper.properties.in === 'query') {
+            generatedStore.collection.auth = {};
+          }
+        }
+        generatedStore.collection.auth = authHelper;
```

Comment:
This will be executed regardless of line 260 🤔 
Can you also add a regression test for this specific case? There should be a case that tests for collection.auth eing empty if in='header'/'query'

Also, why are we doing this in the first place? Why can't apiKey auths be left in the collection.auth property?

Reply:
I will make the changes and add the test case. 

Adding apiKey to collection.auth property which is not recognized as one of the known formats results in an empty selection in the collection level auth dropdown in Postman app. It doesn't default to 'noauth'.  
<img width="755" alt="Screenshot 2020-05-26 at 1 49 53 PM" src="https://user-images.githubusercontent.com/31915995/82877514-e7506c00-9f57-11ea-86f9-0b4913f6d62d.png">

Reply:
Oops! Made changes. @abhijitkane

Reply:
@abhijitkane 
if type is `apikey`, but `in` isn't header/query - we should add the auth to collection. So I have made the required changes. Only in case of api-key and header/query the collection level auth will be 'noauth'. Rest of the cases we will add it directly.